src/sentry/static/sentry/app/views/settings/incidentRules/create.tsx

```diff
@@ -7,15 +7,21 @@ import recreateRoute from 'app/utils/recreateRoute';
 
 import RuleForm from './ruleForm';
 
+type RouteParams = {
+  orgId: string;
+  projectId: string;
+  incidentRuleId: string; //TODO: make optional
```

Comment:
We don't get `incidentRuleId` when creating a new field but making this optional breaks things with child components.

Reply:
Let's make this `TODO(ts)` so we can find it easier later